cg/services/fastq_concatenation_service/utils.py

```diff
@@ -73,7 +73,11 @@ def sort_files_by_name(files: list[Path]) -> list[Path]:
 
 
 def file_can_be_removed(file: Path, forward_file: Path, reverse_file: Path) -> bool:
-    return file.suffix == FileExtensions.GZIP and file != forward_file and file != reverse_file
```

Comment:
This was a check that would not work in a directory with mixed files.

Reply:
Hmm, I don't quite understand? What does your change allow that was not allowed previously?

Reply:
We now put all sample related files into the same directory PRIOR to concatenating.

Removing all files with only the ".gz" suffix would also remove:
vcf.gz
any_other_file.gz

we don't want that

Reply:
Ah, I see 👍